### ISSUE TITLE
turn off word wrap script output

### ISSUE THREAD

**Issue by null338** [bugy/script-server#471]
Hello,

Is there is a setting to turn off word wrap on the script output? My script output has very log lines and the script window appears to be truncating to the next line. Even when I use the copy button and paste to an editor the the lines are split up. 

 It would be helpful if I could find a way to scroll the window from left to right to keep the lines intact.  

Also, is there a way to change the theme in the 'latest' build? The documentation suggest there is, however, I am unable to get it to change to the dark theme. 

Thank you!

**Comment by bugy** [comment]
Hi @null338, unfortunately there is no such feature :(

Regarding the theme: did you try adding `conf/theme/theme.css` file? With proper variables (you could get one from samples folder)

**Comment by null338** [comment]
Hi @bugy ,

Yes, I've tried that. The runners work, so I believe I have it setup correctly.

```
WORKDIR /app

./conf/
├── conf.json
├── logging.json
├── runners
│   ├── job1.json
│   ├── job2.json
│   └── job3.json
└── theme
    ├── darkBackground_header.jpg
    ├── darkBackground_login.jpg
    └── theme.css

```

**Comment by bugy** [comment]
Probably you are using released version. Please try dev one instead:
https://github.com/bugy/script-server/releases/tag/dev

**Comment by null338** [comment]
Yes. Will the themes be released to the 'released' 'latest' version?

**Comment by bugy** [comment]
Yes. Unfortunately I cannot find time lately to finish the full testing and do the release
